elide-spring/elide-spring-boot-autoconfigure/src/test/resources/application.yaml

```diff
@@ -13,6 +13,13 @@ elide:
     path: /doc
     enabled: true
     version: "1.0"
+  async:
+    enabled: true
```

Comment:
We need a URL path for this as well.   Async should probably inherit from the other endpoint configs.

Reply:
As discussed we will leave this param as is 

> The url param does exist but I have not set it specifically in application.yaml because the swagger documentation for async models is available at /doc, and the models are also accessible via jsonapi and graphgl so i did not set it specifically to something there.